101-webapp-linux-managed-mysql/azuredeploy.json

```diff
@@ -99,19 +100,41 @@
             "type": "string",
             "defaultValue": "Gen5",
             "metadata": {
-                "description": "Azure database for PostgreSQL sku family"
+                "description": "Azure database for mySQL sku family"
             }
         }
     },
     "variables": {
-        "databaseName": "[concat(parameters('siteName'), 'database')]",
-        "serverName": "[concat(parameters('siteName'), 'mysqlserver')]",
-        "hostingPlanName": "[concat(parameters('siteName'), 'serviceplan')]"
+        "databaseName": "[concat('database', uniqueString(resourceGroup().name))]",
+        "serverName": "[concat('mysql-',  uniqueString(resourceGroup().name))]",
```

Comment:
this needs to be globally unique, so use resourceGroup().id

Reply:
Thanks for the feedback @bmoore-msft 

Done.